todo/src/actions/index.js

```diff
@@ -0,0 +1,11 @@
+export const ADD_TODO = 'ADD_TODO';
```

Comment:
nice job on your actions.  its generally best practices to break the payload on the next line for readability,  but its set up correctly.